package.json

```diff
@@ -18,22 +18,22 @@
   },
   "homepage": "https://github.com/cliqz-oss/android-browser#readme",
   "dependencies": {
-    "@cliqz/indexeddbshim": "3.4.1",
-    "browser-core": "http://cdn2.cliqz.com/update/edge/react-native/v3.26/3.26.2.tgz",
+    "@cliqz/indexeddbshim": "^4.1.0",
```

Comment:
TODO updated to latest @cliqz/indexeddbshim 4.1.1 once npm packge is publish to registry

Reply:
Done